bot.js

```diff
@@ -61,6 +68,15 @@ client.on('message', msg => {
         } else {
             // By default, it's assumed that player is not in list
             let playerInList = false;
+
+            // Checks if player is in list
+            for (i=0; i<playerList.users.length; i++) {
+                if (playerList.users[i].name == username[1]) {
+                    playerInList = true;
+                }
+            }//MOVED THIS HERE TO KEEP THE CHECKING IF THE PLAYER IS IN THE LIST IN THE SAME SPOT
+            //INSTEAD OF SAYING, THEY AREN'T [DO OTHER STUFF] WAIT, ARE THEY IN THE LIST?  IT JUST MAKES MORE SENSE THIS WAY
+
```

Comment:
```suggestion

            // Checks if player is in list
            for (i=0; i<playerList.users.length; i++) {
                if (playerList.users[i].name == username[1]) {
                    playerInList = true;
                }
            }//MOVED THIS HERE TO KEEP THE CHECKING IF THE PLAYER IS IN THE LIST IN THE SAME SPOT
            //INSTEAD OF SAYING, THEY AREN'T [DO OTHER STUFF] WAIT, ARE THEY IN THE LIST?  IT JUST MAKES MORE SENSE THIS WAY

```